client/src/redux/slices/language.slice.js

```diff
@@ -83,6 +83,77 @@ const languageSlice = createSlice({
       )
       state.lessonData.vocab[vocabIndex] = payload.vocab
     },
+    patchSelectedLesson: (state, { payload }) => {
+      /* Patches the fields for a selected lesson. This is called after submitting on <UpdateLesson/>. */
+
+      // Update the data in allLessons
+      const lessonIndex = state.allLessons.findIndex(
+        (element) => element._id === state.currentLessonId,
+      )
+
```

Comment:
Take a look at this: https://stackoverflow.com/questions/43807515/eslint-doesnt-allow-for-in

I think ESLint wants us to do something like:

```
for (var key of Object.keys(p)) {
    // Add logic here
}
```